counter.go

```diff
@@ -188,49 +176,40 @@ func (c *prefixed) Prefix() string {
 }
 
 // GetCounter returns a counter of given name, if doesn't exist than create.
-func (c *CounterBox) GetCounter(name string) Counter {
-	c.m.RLock()
-	v, ok := c.counters[name]
-	c.m.RUnlock()
+func (c *CounterBox) GetCounter(name string) (v Counter) {
+	value, ok := c.counters.Load(name)
```

Comment:
This introduces a race condition. It may happen:
 1. two routines calls `Load()` same time. 
 2. Routine 1 gets null
 3. Routine 2 gets null
 4. Routine 1 stores value
 5. Routine 2 also stores the value

This would lead to one of the Counters to be returned and modified and afterward removed by a GC.

You should use `Map.LoadOrStore` on line 186 to prevent it.